### DISCUSSION TITLE
404 error after page refresh

### DISCUSSION THREAD

**Discussion by kaleshasheik** [vercel/next.js#13754] [Help]
Hi ,

 I have below to redirect to another page when button was clicked in my React.js application.

import Router from 'next/router'
Router.push(`/dashboard?customerId=${customerId}`)

After button was clicked, it was redirected to dashboard page. But , when i do page refresh, I am getting 404 error. Any suggestion to fix this ?

**Comment by jamesmosier** [comment]
Hi @kaleshasheik, do you happen to have a repository showing this issue?

**Reply by kaleshasheik** [reply]
> Hi @kaleshasheik, do you happen to have a repository showing this issue?

Hi James,
 
   I am facing this issue in development environment.  Locally,  everything looks good.

**Reply by jamesmosier** [reply]
These are incredibly difficult to help with without a repository where this can be reproduced. It could be anything from the setup in your repository to your proxy to your server and anything and everything in between.

Do you have code you could share?

**Reply by jamesmosier** [reply]
Comparing the [2 discussions](https://github.com/vercel/next.js/discussions/13792) you opened, you may have an issue with case sensitivity in your routes.

Try to normalize all of your routes to `dashboard` instead of a mix of `dashboard` and `dashBoard`. Also ensure you name your page `pages/dashboard`

**Reply by ShafSpecs** [reply]
> Comparing the [2 discussions](https://github.com/vercel/next.js/discussions/13792) you opened, you may have an issue with case sensitivity in your routes.
> 
> Try to normalize all of your routes to `dashboard` instead of a mix of `dashboard` and `dashBoard`. Also ensure you name your page `pages/dashboard`

I face the same issue, this is the hosted [site](https://openlibrary-ui-kit.vercel.app). 
Click 'Browse Library, then hit refresh.

The repo is ShafSpecs/Open-Library-UI-Kit
